Agoda.Frameworks.Grpc/Agoda.Frameworks.Grpc.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageProjectUrl>https://github.com/agoda-com/net-loadbalancing</PackageProjectUrl>
+    <Authors>Agoda</Authors>
+    <Product>Agoda.Frameworks.Grpc</Product>
+    <Company>Agoda</Company>
+    <Description>Agoda.Frameworks.Grpc</Description>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OS)'!='Windows_NT'">
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)'=='Windows_NT'">
+    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
```

Comment:
We might remove net462 support in the future. Could you please remove it and see if it works?

Reply:
Tested it. It works